demo/src/app.jsx

```diff
@@ -888,6 +879,14 @@ function AppContent({ language, onLanguageClick }) {
     );
 }
 
+const WS_URL = 'WS_URL';
+const WS_URL_KEY = 'WS_URL_KEY';
+function AppWSConsumer() {
+    useListener(WS_URL_KEY, (event) => {
+        console.log('WS called UPDATE_FINISHED : ', event);
+    });
```

Comment:
Where do you simulate a notification event for `WS_URL_KEY` in the demo ?

Reply:
remove from demo.